### ISSUE TITLE
Can not build docker dev image on Windows

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-online-tools#353]
Running `docker-compose -f docker-compose.dev.yml up --build` in Windows with Docker for Windows version 4.7 generates the following error:

```
 > [stage-1 2/6] COPY --from=python-container / /:
------
cannot copy to non-directory: /var/lib/docker/overlay2/vd0vu5ud51gejfddtws3ygy9j/merged/usr/share/doc/perl-base
ERROR: Service 'spdx' failed to build : Build failed
```

I found [this stack overflow article](https://stackoverflow.com/questions/68075608/docker-cannot-copy-to-non-directory-var-lib-docker-overlay2-xw77p2bxfkhhnwqs) which describes a similar issue - there is no resolution as yet.

**Comment by goneall** [comment]
@rtgdk Can you try creating the docker image on you Linux machine and let me know if it succeeds?

**Comment by goneall** [comment]
@Ugtan Any ideas on a workaround?

**Comment by goneall** [comment]
From reading some of the other posts, it looks like the recommended work around is to specify the specific directories to be copied into.

However, the directory that fails is pretty deeply nested, so I doubt that workaround would work.

**Comment by goneall** [comment]
Found the issue - turns out OpenJDK:8 has a symbolic link for the file which can not be overriden by the copy.

Using OpenJDK:slim does not have these directories.  I'll create a PR.

**Comment by goneall** [comment]
Resolved with PR #356
